atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweeperService.java

```diff
@@ -15,47 +15,53 @@
  */
 package com.palantir.atlasdb.sweep;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import java.util.Optional;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DefaultValue;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 
+import com.palantir.logsafe.Safe;
+
 /**
  * Provides endpoints for sweeping a specific table.
  */
 @Path("/sweep")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
 public interface SweeperService {
-    /**
-     * Sweep a particular table from EMPTY startRow with default {@link SweepBatchConfig}.
-     */
-    @POST
-    @Path("sweep-table")
-    @Produces(MediaType.APPLICATION_JSON)
-    void sweepTable(@QueryParam("tablename") String tableName);
 
-    /**
-     * Sweep a particular table from specified startRow with default {@link SweepBatchConfig}.
-     */
-    @POST
-    @Path("sweep-table-from-row")
-    @Produces(MediaType.APPLICATION_JSON)
-    void sweepTableFromStartRow(
-            @QueryParam("tablename") String tableName,
-            @Nonnull @QueryParam("startRow") String startRow);
+    default SweepTableResponse sweepTable(String tableName) {
```

Comment:
should we get rid of these APIs as we are no longer exposing the endpoints. This is technically a break, but probably noone will be affected.

Reply:
i mainly just added these for easier usage in java (otherwise you have to pass in 5 `Optional.empty()`s